### ISSUE TITLE
[DropDownList] Incorrect aria-required rendering when using the label property

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-vue#441]
**Describe the bug**
When using the DropDownList in combination with its **label** and **aria-required** props the aria-required is not correctly rendered

**To Reproduce**
1. Open [this CodeSandbox project](https://codesandbox.io/s/crazy-cache-vrm9vx?file=/src/SelectListWrapper.vue)
2. Open the browser's DevTools 
3. Inspect the DropDownlist
4. See where the **aria-required** attribute is rendered

**Expected behavior**
The **aria-required** attribute should be rendered on the element with the "combobox" role.

**Screenshots**
<img width="791" alt="image" src="https://github.com/telerik/kendo-vue/assets/41293735/bf4d8fcd-7496-4b76-8ec6-ae9dbb05d64f">
